CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### pack, serve
 
+* Fixed `enact serve` fails with multiple ports by overriding resolvePort as devServer port.
```

Comment:
Please move this under ### serve section.

Reply:
OK